WalletWasabi.Gui/ViewModels/AddressViewModel.cs

```diff
@@ -42,28 +40,37 @@ public AddressViewModel(HdPubKey model, Global global)
 				.ObserveOn(RxApp.TaskpoolScheduler)
 				.Where(x => x)
 				.Take(1)
-				.Subscribe(_ =>
+				.Select(x =>
+				{
+					var encoder = new QrEncoder();
+					encoder.TryEncode(Address, out var qrCode);
+					return qrCode.Matrix.InternalArray;
+				})
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(qr =>
 				{
 					try
```

Comment:
I wonder if we could remove the try/catch block altogether since we have an `onError` listener already?

Reply:
Yes, the `try-catch` can be removed. I can confirm the `onError` catches the exceptions.

Reply:
Removed.